lib/tlsca/ca.go

```diff
@@ -474,6 +476,10 @@ var (
 	// LicenseOID is an extension OID signaling the license type of Teleport build.
 	// It should take values "oss" or "ent" (the values returned by modules.GetModules().BuildType())
 	LicenseOID = asn1.ObjectIdentifier{1, 3, 9999, 2, 14}
+
+	// PinnedIPASN1ExtensionOID is an extension ID used when encoding/decoding
+	// the IP the certificate is pinned to.
+	PinnedIPASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 2, 15}
```

Comment:
Please ensure the RFD is updated to match this ASN1 Object Identifier after this is merged.

Reply:
Yep 👍